docs/openfaas-pro/retries.md

```diff
@@ -32,6 +32,37 @@ Timings are specified in Golang durations such as `10s` or `1m`.
 * `maxRetryAttempts` is the amount of times to try sending a message to a function before discarding it.
 * `httpRetryCodes` is a comma-separated list of HTTP status codes which the queue worker will retry when received from a function.
 
+## Configure retries per function
+
+The retry configuration can be overridden on a per function basis using annotations on the function:
+
+=== "CLI"
+
+    ```bash
+    faas-cli store deploy chaos \
+      --annotation com.openfaas.retry.attempts=30 \
+      --annotation com.openfaas.retry.codes=429 \
+      --annotation com.openfaas.retry.min_wait=5s \
+      --annotation com.openfaas.retry.max_wait=1m
+    ```
+
+=== "stack.yml"
```

Comment:
stack.yml should be first, since most of our users don't deploy with CLI flags, but with YAML. I know we like to do so for testing.

Reply:
The stack.yml is now displayed first.